src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -62,6 +72,11 @@ public BindingWorkflow(IHost host, ProjectInformation project)
 
         #region Workflow state
 
+        public List<Project> BindingProjects
```

Comment:
Try if the default is good enough. Probably safe to assume that the instance based approach will work in this case.